django/db/backends/base/operations.py

```diff
@@ -45,6 +45,9 @@ class BaseDatabaseOperations:
     UNBOUNDED_FOLLOWING = 'UNBOUNDED ' + FOLLOWING
     CURRENT_ROW = 'CURRENT ROW'
 
+    # Prefix for EXPLAIN queries. Empty string if the backend does not support this.
```

Comment:
Would `None` be more semantic than an empty string? Then `supports_explaining_query_execution` could check `is not None`.